config/routes.rb

```diff
@@ -121,9 +121,13 @@
     resource :hca_attachments, only: :create
     resource :form1010_ezr_attachments, only: :create
 
-    resources :caregivers_assistance_claims, only: :create
-    post 'caregivers_assistance_claims/download_pdf', to: 'caregivers_assistance_claims#download_pdf'
-
+    resources :caregivers_assistance_claims, only: :create do
+      collection do
+        get(:facilities)
```

Comment:
Added new `facilities` route, and moved the post declaration here. This is functionally the same as before, just easier to read.